src/api/shopper-insights/validation.js

```diff
@@ -1,5 +1,6 @@
 /* @flow */
 
+// eslint-disable-next-line import/no-deprecated
```

Comment:
this is fixed in https://github.com/paypal/paypal-checkout-components/pull/2332 but that can't be merged until the client-id error is fixed (in this PR)